ADR/ADR009-replatforming-to-ECS.md

```diff
@@ -0,0 +1,38 @@
+# ADR009: Re-platforming GOV.UK Forms to AWS ECS Fargate
+
+Date: 2022-10-27
+
+## Status
+
+Accepted
+
+## Context
+
+GOV.UK Forms currently runs on GOV.UK PaaS which is being decommissioned in
+December 2023 and all tenants must re-platform.
+
+## Decision
+
+Move GOV.UK Forms onto AWS Elastic Container Service (ECS) with Fargate and supporting
+AWS services such as RDS and Elasticache. All infrastructure to be managed with
+Terraform. Continuous integration and deployment patterns will be implemented
+(technology choices are TBD).
+
+## Consequences
+
```

Comment:
- The effort required to design and migrate to new infrastructure will reduce the amount of roadmap features that can be developed during the migration.